service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2610,9 +2610,8 @@ components:
     ExportResult:
       type: object
       properties:
-        status:
-          type: string
-          enum: ["RUNNING", "COMPLETE"]
+        isSuccessful:
```

Comment:
While not currently needed, changing this to a boolean is potentially limiting in the future. Maybe just change the values to ["SUCCEEDED", "FAILED"]?

Reply:
OK with me. I'll do this in the other PR where I updated the API -- this change got mixed up between the two.